providers/user.rb

```diff
@@ -177,7 +177,7 @@ def user_has_permissions?(name, vhost, perm_list = nil)
 
 action :change_password do
   if user_exists?(new_resource.user)
-    cmd = "rabbitmqctl -q change_password #{new_resource.user} #{new_resource.password}"
+    cmd = "rabbitmqctl -q change_password #{new_resource.user} \"#{new_resource.password}\""
```

Comment:
Quoting the password here feels like the right move, but I'm pretty sure this would fail if the password contains a double-quote character.  Might be best to define a helper function to escape certain characters in `new_resource.password` for use by the shell

Reply:
Pretty much any such function would have limitations, as do all quoting mechanisms. I personally don't see double quotes used in generated passwords very often while ampersands are common in my experience.

Is a helper function worth the complexity? 🤔

Reply:
I realized that in the add user op, this scenario was already handled by using single quotes & escaping single quotes in the new_resource.password variable, I duplicated that functionality in change password.

Reply:
I've seen quoting in the random passwords generated by 1Password.  The previous form of this command should've failed with any quotes in the password, so it's probably not worth an escaping handler at this point.